commonmark/src/main/java/org/commonmark/internal/inline/Scanner.java

```diff
@@ -0,0 +1,115 @@
+package org.commonmark.internal.inline;
+
+import org.commonmark.internal.util.CharMatcher;
+import org.commonmark.internal.util.Parsing;
+
+public class Scanner {
```

Comment:
This scanner class helped a lot the code readability. :)

Reply:
Yeah, I think so too :).